tests/Mantis/UserTest.php

```diff
@@ -0,0 +1,144 @@
+<?php
+# MantisBT - A PHP based bugtracking system
+
+# MantisBT is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# MantisBT is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with MantisBT.  If not, see <http://www.gnu.org/licenses/>.
+
+/**
+ * Test cases for User API within mantis
+ *
+ * @package    Tests
+ * @subpackage UserAPI
+ * @copyright  Copyright 2023  MantisBT Team - mantisbt-dev@lists.sourceforge.net
+ * @link       http://www.mantisbt.org
+ *
+ * @noinspection PhpIllegalPsrClassPathInspection
+ */
+
+# Includes
+use Mantis\Exceptions\ClientException;
+
+require_once 'MantisCoreBase.php';
+
+/**
+ * PHPUnit tests for User API
+ */
+class MantisUserApiTest extends MantisCoreBase {
+
+	const TEST_EMAIL = 'test@uniqueness.test';
+
+	protected static $user_id;
+
+	public static function setUpBeforeClass() {
+		parent::setUpBeforeClass();
+
+		$t_cookie = user_create(
+			'User' . rand(),
+			'password',
+			self::TEST_EMAIL
+		);
+		/** @noinspection PhpUnhandledExceptionInspection */
+		self::$user_id = user_get_id_by_cookie( $t_cookie );
+	}
+
+	public static function tearDownAfterClass() {
```

Comment:
Should you call `parent::tearDownAfterClass()` similar to calling `parent::setUpBeforeClass()` above?

Reply:
Technically yes I guess, but the method is not defined in the parent, so it would just be unnecessary overhead